src/parcsr_mv/par_csr_matrix.h

```diff
@@ -30,6 +30,7 @@ typedef struct hypre_ParCSRMatrix_struct
 
    HYPRE_BigInt          global_num_rows;
    HYPRE_BigInt          global_num_cols;
+   HYPRE_Int             global_num_rownnz; /* global number of nonzero rows */
```

Comment:
No sure why we need this global information and where it is used. If it's needed, it may be a `HYPRE_BigInt`.

Reply:
This isn't necessary either. Thanks again for catching!

Reply:
I realized this is actually needed by SSAMG when it computes some statistics. So, I'm adding this back along with a function to compute the global number of nonzero rows in 972376c. Note that you'll not see this being used in the current branch, however it's needed in recmat.

Reply:
So, it needs to be `BigInt`?

Reply:
Correct! I set it as `BigInt` in abfae37.